pkg/cmd/extension/manager.go

```diff
@@ -249,6 +250,64 @@ func (m *Manager) installDir() string {
 	return filepath.Join(m.dataDir(), "extensions")
 }
 
+func (m *Manager) Create(name string) error {
+	exe, err := m.lookPath("git")
+	if err != nil {
+		return err
+	}
+
+	err = os.Mkdir(name, 0755)
+	if err != nil {
+		return err
+	}
+
+	initCmd := m.newCommand(exe, "init", "-b", "main", "--quiet", name)
```

Comment:
Let's not use `-b main` here and respect the person's git settings for default main branch.
```suggestion
	initCmd := m.newCommand(exe, "init", "--quiet", name)
```
We suggest using `-b main` on the docs site because the instructions there are manual; thus, a person following them can change them at invocation time.